package.json

```diff
@@ -52,7 +52,9 @@
   "devDependencies": {
     "@govflanders/vl-ui-input-addon": "^3.9.1",
     "@govflanders/vl-ui-util": "^3.9.1",
+    "@govflanders/vl-ui-icon": "3.9.1",
```

Comment:
Deze dependency lijkt me niet nodig, wordt namelijk niets van gebruikt.

Reply:
ah jawel, in het component object. zie mijn vraag hierboven.

Reply:
Nee, dat is een dependency op onze vl-ui-icon, niet op die van @govflanders.

Reply:
ah just, maar dan moet er toch wel een zijn op vl-ui-icon in package.json?

Reply:
Ja, die werd in deze PR ook toegevoegd door @wautergu.